### PR TITLE
add descriptive research

### PR DESCRIPTION
**Pull request by claireb-gsa**
Adding section on descriptive research to basics, creating consistent pattern of what questions research helps ask, shifting language from I to we.

### CODE REVIEW

File: _pages/research/clarify-the-basics.md [hunk 1]
```diff
@@ -47,28 +47,37 @@ Because a collaborative approach increases the team’s overall empathy and effi
 
 
 ## Research types
-On any given project you should only include the research activities that will inform the decisions you plan to make. Broadly speaking, 18F research falls into three categories:
+On any given project you should only include the research activities that will inform the decisions you plan to make. Broadly speaking, 18F research falls into several categories that can done separately or in a sequence as you focus from problem definition to solution exploration:
```
Comment (MelissaBraxton) [inline _pages/research/clarify-the-basics.md:50]: Wondering whether we need the last clause?  If we want to keep it, consider making it active voice.

Suggested edit: 

Broadly speaking, 18F research falls into several categories. You can do research from any category on its own or in a sequence as you focus from problem definition to solution exploration:

**Review by MelissaBraxton — COMMENTED** [review]
Left one comment, but otherwise LGTM! I'll leave it to @bpdesigns for final review/merge.

File: _pages/research/clarify-the-basics.md (hunk 1 above)
Comment (bpdesigns) [reply _pages/research/clarify-the-basics.md:50]: I'd prefer to keep that content as a way to connect the research types. I do like your update and active voice language @MelissaBraxton 

@claireb-gsa approved with this update.

**Review by bpdesigns — APPROVED** [review]
LGTM

**Pull request merged**